File: src/providers/keys.ts
// API 密钥：仅读取构建时注入的环境变量（.env / 部署平台的环境变量）。
// 应用内不再提供密钥录入界面；需要密钥的信源在构建时配置好对应 VITE_* 即可启用，
// 未配置则该信源自动不参与对比。

export type KeyId = 'qweather' | 'caiyun' | 'owm' | 'weatherapi' | 'iqair' | 'waqi'

const ENV_KEYS: Record<KeyId, string | undefined> = {
  qweather: import.meta.env.VITE_QWEATHER_KEY as string | undefined,
  caiyun: import.meta.env.VITE_CAIYUN_KEY as string | undefined,
  owm: import.meta.env.VITE_OWM_KEY as string | undefined,
  weatherapi: import.meta.env.VITE_WEATHERAPI_KEY as string | undefined,
  iqair: import.meta.env.VITE_IQAIR_KEY as string | undefined,
  waqi: import.meta.env.VITE_WAQI_TOKEN as string | undefined,
}

export function getKey(id: KeyId): string | undefined {
  return ENV_KEYS[id]?.trim() || undefined
}

File: ios/Sources/Utils/Keys.swift
import Foundation

// 可选信源 API key（和风 / 彩云 / OWM）。番禺核心源（中央气象台 / 番禺气象台 / AQI）
// 全部原生直抓，无需任何 key 或后端。填了 key 就多一个对比源，不填则自动跳过。
//
// 个人自用：直接在下面硬编码即可（建议把本文件加入 .gitignore 防止泄漏）。
enum Keys {
    static let qweather = ""
    static let caiyun   = ""
    static let owm      = ""
}
